WooCommerce/Classes/ViewModels/Order Details/Refunded Products/RefundedProductsDataSource.swift

```diff
@@ -7,9 +7,9 @@ import Yosemite
 /// Must conform to NSObject so it can be the UITableViewDataSource.
 ///
 final class RefundedProductsDataSource: NSObject {
-    /// Refunds
+    /// Aggregate data for refunded products
     ///
-    private var items: [OrderItemRefundSummary]
+    private(set) var refundedProducts: [AggregateOrderItem]
```

Comment:
nit: could this be `private let` since it doesn't seem to be modified after init?